### ISSUE TITLE
75629493 Unable to load new email address (Wales)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#963]
Issue reported by **Admin-Wales** at 2017-05-25 08:11:33 UTC
Time: 2017-05-25T08:09:49+00:00
Session ID: 05fa1f8ee4748b95b3e5a102a59b70dd
Problem Page URL: [/userid_details/5462b41ee937907d70000503](/userid_details/5462b41ee937907d70000503)
Previous Page URL: [https://www.freereg.org.uk/userid_details/5462b41ee937907d70000503](https://www.freereg.org.uk/userid_details/5462b41ee937907d70000503)
Reported Issue:
Trying to update a transcribers email address when this error is generated. Tried to do this several ways, inc clicking yes to email is valid (on both new and old email addresses). Changed an email address last week so unsure of what is different this week.    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/592691b5f493fd763208e4e1/_old_)

**Comment by ghost** [comment]
![dai](https://cloud.githubusercontent.com/assets/20557838/26493269/6f6c8a9a-4210-11e7-94db-7de542addea5.png)

**Comment by richpomfret** [comment]
@SteveBiggs Still an issue?

**Comment by SteveBiggs** [comment]
Yes this was still an issue. I checked in Refinery and it does indeed show David's email address as daiandann@gmail.com:
![capture](https://user-images.githubusercontent.com/25303607/29408099-ce88ca1a-833e-11e7-9298-ae827957bad7.jpg)

However, in his profile, it shows the fsnet address:
![capture](https://user-images.githubusercontent.com/25303607/29408168-18565856-833f-11e7-9a53-f55b23432100.jpg)
So I changed the Refinery address to the fsnet one, then edited his profile to the new gmail address and that was now successful.

Not sure why Refinery was out of sync with the profile, but all is good now.
